docs/developer/contributing/api-guidelines.rst

```diff
@@ -226,8 +226,8 @@ response indicating:
 - next page ``rel="next"``;
 - last page ``rel="last"``.
 
-Link headers are described by `RFC 5988
-<https://tools.ietf.org/html/rfc5988#page-6>`_.
+Link headers are described by `RFC 8288
+<https://tools.ietf.org/html/rfc8288#page-6>`_.
```

Comment:
I see another remaining link to page 6 which is not of particular interest anymore.

Reply:
@ableuler: fixed?